win/pkgs/vsbuildtools.ps1

```diff
@@ -0,0 +1,28 @@
+#Requires -RunAsAdministrator
+
+$ErrorActionPreference="Stop"
+
+$vs_where = "${Env:ProgramFiles(x86)}/Microsoft Visual Studio/Installer/vswhere.exe"
+if (Test-Path $vs_where)
+{
+    $vs_home = & $vs_where -latest -products * -property installationPath
+    if ($vs_home)
+    {
+        Write-Host "Found existing VS installtaion under `"$vs_home`". Skip installation."
+        exit 0
+    }
+}
+
+Invoke-WebRequest -Uri https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile "${Env:SCRATCH}/vs_buildtools.exe"
+
+$install_cmd = "`"${Env:SCRATCH}/vs_buildtools.exe`" --quiet --wait --norestart --nocache " `
+    + "--add Microsoft.VisualStudio.Workload.VCTools;includeRecommended "                   `
+    + "--add Microsoft.VisualStudio.Component.Windows10SDK.16299 "                          `
+    + "--installPath C:\BuildTools || exit /b %ERRORLEVEL%"
+cmd /c $install_cmd
```

Comment:
Why do you need to `cmd /c`?
Should it be `& xxxx | Out-Null`?

Reply:
hmm, good question.

I had an impression that without calling it in cmd, the `-wait` switch doesn't work. And it looks wierd. I'll double confirm it later.

Reply:
I recalled why i use `cmd/ c` here.

I want to exit with `%ERRORLEVEL%` on non-zero return code.